spec/models/user_spec.rb

```diff
@@ -87,13 +87,62 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before do
+          registration = create(:registration, user: user, conference: conference)
+          create(:events_registration, event: event1, registration: registration, attended: true)
```

Comment:
Same here, although this one needs to be called in the before, to create it. So:

````Ruby
 describe '#attended_event?' do
  let(:registration) { :create(:registration, user: user, conference: conference) }
  let(:events_registration) { create(:events_registration, event: event1, registration: registration, attended: true) }
  
  context 'user has attended to the event' do
    before do
      events_registration
    end